environments/icds-cas/app-processes.yml

```diff
@@ -44,7 +44,7 @@ celery_processes:
   'celery1':
     ucr_queue:
       concurrency: 4
-      max_tasks_per_child: 5
+      max_tasks_per_child: 1
```

Comment:
why this change?

Reply:
It's not a necessary change, but left a comment about why in the commit https://github.com/dimagi/commcare-cloud/pull/2062/commits/1c7ee7d78407bea740303610bd0fe2e4a635b858

> There isn't a lot of activity on these queues and it's likely a high risk for memory leaks due to loading many documents